invenio_app_ils/permissions.py

```diff
@@ -38,10 +38,12 @@ def backoffice_permission(*args, **kwargs):
 class LoanOwnerPermission(Permission):
     """Return Permission to evaluate if the current user owns the loan."""
 
-    def __init__(self, record):
+    def __init__(self, record, patron_pid=None):
         """Constructor."""
+        if record:
```

Comment:
did you have a case that record is not passed here other than the newly created factory? The class was specifically meant for Loan's protection while the circulation_status is a resolved reference. I think this should fail when the record is not provided, especially because we use it in the config `read_permission_factory_imp=LoanOwnerPermission`, instead I would just write simple permission factory just for the `circulation_status`.

Reply:
No, it's only the new one that uses it. I wanted to re-use this since I needed a permission check  for loan ownership and this one already does it.

`LoanOwnerPermission` doesn't use `record` besides for getting the patron pid so what do you think about changing the constructor to take the patron pid instead of the record?